app/components/AuctionHighestBid.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import { inject, observer } from "mobx-react";
+import { computed } from "mobx";
+import BigNumber from "bignumber.js";
+import { colors } from "../styles";
+import styled from "react-emotion";
+
+const Title = styled("div")`
```

Comment:
These two seem to be shared a couple times between component. Maybe put into a shared `auction.js` file

Reply:
Will add to `Auction.js`